x/genaccounts/server/cli/genesis_accts.go

```diff
@@ -1,20 +1,26 @@
-package init
+package cli
```

Comment:
This should live under `x/genaccounts/client/cli/genesis_accts.go`, not `server`.

Reply:
hmm this is used by `gaiad` client - it would be nice to distinguish from all the other modules where all `client` gets attached to `gaiacli` any other suggestions besides just `client/` ? maybe `server_client`? so long though

Reply:
Can't we just leave all of that in the `server` package?

Reply:
@rigelrozanski I believe it should live under `x/genaccounts/client/cli` because it's a client command implementation. Yes, I know we have commands mounted under `gaiad` and `gaiacli` respectively, but having their import paths as a means of disambiguation isn't too helpful. 

If you're fully against this still for whatever reason, I suppose `server_client` is a happy compromise :)

Reply:
Let's avoid dropping underscores in our package names please: https://blog.golang.org/package-names

Reply:
alright moved `server/` ->  `client/` for genutil and genaccounts